### PR TITLE
feat: Add new sections to '/containers/chiselled'

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Add new sections to '/containers/chiselled' based on [copydoc](https://docs.google.com/document/d/1ZkQ-UFTwsPkMnQn8IEEjZOXMy40tkgBscw_-qq9K3nc/edit) and [design](https://www.figma.com/design/YfNc8KmYqa2e7JgbFo6lXa/24.04-U.com---%2Fcontainers?node-id=440-3441&t=Snw41ddNHSYNRMvI-0)

## QA

- Go to https://ubuntu-com-13991.demos.haus/containers/chiselled
- Check content matches the [copydoc](https://docs.google.com/document/d/1ZkQ-UFTwsPkMnQn8IEEjZOXMy40tkgBscw_-qq9K3nc/edit)
- Check design matches the [design](https://www.figma.com/design/YfNc8KmYqa2e7JgbFo6lXa/24.04-U.com---%2Fcontainers?node-id=440-3441&t=Snw41ddNHSYNRMvI-0)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12561

### CODE REVIEW

**Comment by petesfrench** [comment]
@Sophie-32 Can you take a quick look at this please?

File: templates/containers/chiselled/index.html [hunk 1]
```diff
@@ -1,515 +1,605 @@
...
+  <hr class="is-fixed-width" />
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <h2>
+        <a href="/containers/contact-us" class="js-invoke-modal">Get a chiselled container built by Canonical&nbsp;&rsaquo;</a>
+      </h2>
     </div>
-  </div>
-</section>
+  </section>
```
Comment (petesfrench) [inline templates/containers/chiselled/index.html:70]: change (section added)

File: templates/containers/chiselled/index.html [hunk 2]
```diff
@@ -1,515 +1,605 @@
...
+        <p>
+          <a href="https://github.com/canonical/chisel">Explore Chisel&nbsp;&rsaquo;</a>
+        </p>
       </div>
-      <hr class="p-rule" />
-      <p>Mark Lewis, VP Application Services at Canonical, explains how chisel works.</p>
     </div>
-  </div>
-</section>
+  </section>
```
Comment (petesfrench) [inline templates/containers/chiselled/index.html:94]: change (section added)

**Review by mtruj013 — APPROVED** [review]
LGTM!

**Pull request merged**